### ISSUE TITLE
Table sorting arrow is not shown

### ISSUE THREAD

**Issue by theunreal** [valor-software/ng2-table#228]
Why do the sorting arrow like in the demo is not displayed in my app?

No console errors

**Comment by wilfredmifsud** [comment]
Do you have FontAwesome installed ? As it renders an icon in the th when you sort up/ down (there is no icon when not sorting)

**Comment by Pokom** [comment]
@wilfredmifsud 

It might be worthwhile to consider adding some of these items to the README.md. As awkward as this is to say, it wasn't obvious to me that FontAwesome and Bootstrap we're required to get this to work.

**Comment by georgeedwards** [comment]
Was going to open a new issue/feature request, but it seems like this only works with font-awesome, and the generated icon can't be configured - am I right? If so, I think this would be a really useful feature, to avoid the dependency on bootstrap and font-awesome. I for example, would like the icons to be generated as `<i class="material-icons">arrow_drop_down</i>` and `<i class="material-icons">arrow_drop_up</i>` so I can use Google Icons which I use across my project already.

**Comment by jeff-jacobs** [comment]
I am going to also say the ability to configure the sort icon is kind of a must. I shouldn't have to load the entire font-awesome file just for a couple icons. I built my own font icon library with Fontello for this exact reason, and the fact that I can't use my own icons is kind of a problem.

**Comment by Pokom** [comment]
@jeffaxial 
While this isn't an ideal solution, you can also just use the same classnames as the respective font-awesome icon and have those classes reference your set of icons. I think the true feature request is to have control to configure the icon classes for the table component, which would be ideal in your situation.

**Comment by jeff-jacobs** [comment]
@Pokom Yeah, I ended up adding the fontawesome classes to my fontello icon list classes. It's kind of dirty, but it gets the job done for now. But yes, the real request is to have control over those classes/active states. Thanks :)
